tests/conftest.py

```diff
@@ -41,117 +23,8 @@ def event_loop():
-        dump_coverage(output_dir, files)
-    else:
-        dump_reports(output_dir / worker_id)
-        dump_coverage(output_dir / worker_id, files)
-        if len(os.listdir(output_dir)) == int(os.environ["PYTEST_XDIST_WORKER_COUNT"]):
-            # This is the last teardown of the testsuite, merge the files
-            resources = pd.concat(
-                [pd.read_csv(f) for f in output_dir.glob("**/resources.csv")],
-            )
-            if not resources.empty:
-                resources.sort_values(["n_steps"], ascending=False).to_csv(
-                    output_dir / "resources.csv", index=False
-                )
-            times = pd.concat(
-                [pd.read_csv(f) for f in output_dir.glob("**/times.csv")],
-                ignore_index=True,
-            )
-            if not times.empty:
-                times.sort_values(["duration"], ascending=False).to_csv(
-                    output_dir / "times.csv", index=False
-                )
-
-
-@pytest_asyncio.fixture(scope="session")
-async def eth(starknet: Starknet):
-    return await starknet.deploy(
-        source="./tests/utils/ERC20.cairo",
-        constructor_calldata=[2] * 6,
-        # Uint256(2, 2) tokens to 2
-    )
-
-
-@pytest_asyncio.fixture(scope="session")
-async def account_registry(starknet: Starknet):
-    return await starknet.deploy(
-        source="./src/kakarot/accounts/registry/account_registry.cairo",
-        cairo_path=["src"],
-        disable_hint_validation=True,
-        constructor_calldata=[1],
-    )
-
-
-@pytest_asyncio.fixture(scope="session")
-async def contract_account_class(starknet: Starknet):
-    return await starknet.declare(
-        source="./src/kakarot/accounts/contract/contract_account.cairo",
-        cairo_path=["src"],
-        disable_hint_validation=True,
-    )
-
-
-@pytest_asyncio.fixture(scope="package")
-async def kakarot(
-    starknet: Starknet, eth: StarknetContract, contract_account_class: DeclaredClass
-) -> StarknetContract:
-    return await starknet.deploy(
-        source="./src/kakarot/kakarot.cairo",
-        cairo_path=["src"],
-        disable_hint_validation=True,
-        constructor_calldata=[
-            1,
-            eth.contract_address,
-            contract_account_class.class_hash,
-        ],
-    )
-
-
-@pytest_asyncio.fixture(scope="package", autouse=True)
-async def set_account_registry(
-    kakarot: StarknetContract, account_registry: StarknetContract
-):
-    await account_registry.transfer_ownership(kakarot.contract_address).execute(
-        caller_address=1
-    )
-    await kakarot.set_account_registry(
-        registry_address_=account_registry.contract_address
-    ).execute(caller_address=1)
-    yield
-    await account_registry.transfer_ownership(1).execute(
-        caller_address=kakarot.contract_address
-    )
+pytest_plugins = [
+    "tests.fixtures.1_starknet",
+    "tests.fixtures.2_kakarot",
+    "tests.fixtures.3_account_registry",
```

Comment:
why adding numbering as prefix?

Reply:
I added this to help navigate in the fixtures when reading the file tree, the fixtures order matters because kakarot relies on starknet and account registry on kakarot.

It's a tradeoff for evolution though, especially if we need to add fixtures we may need to rename files, I propose to let this and if there is any issue in evolutivity I remove it